### DISCUSSION TITLE
Encryption type

### DISCUSSION THREAD

**Discussion by gabimoncha** [Tencent/MMKV#830] [Q&A] [answered]
Hello! What type of encryption does this library use? Is there any possibility to change it?

**Comment by lingol** [comment]
Take a look at the FAQ.

**Reply by lingol** [reply]
And no, there's no public interface to change the encryption algorithm. You have to touch the code to do that.

**Comment by gabimoncha** [comment, marked as answer]
@lingol I've been looking into the project and couldn't find it. Can you please link it?

**Reply by lingol** [reply]
https://github.com/Tencent/MMKV/wiki/FAQ
